jest.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  moduleDirectories: ["node_modules", "app/webpacker"],
+  moduleFileExtensions: ["js", "json"],
+  automock: false,
+  resetMocks: true,
+  transform: {
+    "^.+\\.js$": "babel-jest",
+  },
+  cacheDirectory: "<rootDir>/.jest-cache",
+  collectCoverage: true,
+  coverageDirectory: "coverage/frontend",
+  coverageReporters: ["text", "lcov"],
+  collectCoverageFrom: ["<rootDir>/app/webpacker/**/*.js"],
+  reporters: ["default"],
+  setupFilesAfterEnv: ["<rootDir>/spec/javascript/setupTests.js"],
+  transformIgnorePatterns: ["node_modules/*"],
+  moduleNameMapper: {
+    "^@/(.*)$": "<rootDir>/app/javascript/$1",
+  },
+  roots: ["spec/javascript"],
+  testMatch: ["**/spec/javascript/**/*.spec.(js)|**/__tests__/*.(js)"],
```

Comment:
I understand that in rails we keep tests in a seperate `spec` folder but for js unit tests I think its better and easier to spot if test suites are missing if they are in the same directory (Publish and Find both had JS tests in the `app/webpacker` folders

Reply:
removed this change from this pr